### PR TITLE
Description text change for manage policy modal

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
BZ: https://bugzilla.redhat.com/show_bug.cgi?id=2231838

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
image

**Comment by GowthamShanmugam** [comment]
image

**Comment by GowthamShanmugam** [comment]
image

image

image

File: packages/shared/src/table/selectable-table.tsx [hunk 1]
```diff
@@ -97,10 +95,13 @@ export const SelectableTable: SelectableTableProps = <
...
 
-  return loaded && !loadError ? (
-    sortedRows?.length === 0 ? (
-      <Bullseye>{t('Not found')}</Bullseye>
-    ) : (
+  return (
+    <StatusBox
+      loadError={loadError}
+      loaded={loaded}
+      EmptyMsg={emptyRowMessage}
```
Comment (SanjalKatiyar) [inline packages/shared/src/table/selectable-table.tsx:102]:
```suggestion
      EmptyMsg={emptyRowMessage || DefaultEmptyRowMessage}
```

Comment (SanjalKatiyar) [reply packages/shared/src/table/selectable-table.tsx:102]: `const DefaultEmptyRowMessage = () => <Bullseye>{t('Not found')}</Bullseye>`

File: packages/shared/src/table/selectable-table.tsx [hunk 2]
```diff
@@ -97,10 +95,13 @@ export const SelectableTable: SelectableTableProps = <
     columnIndex,
   });
 
-  return loaded && !loadError ? (
-    sortedRows?.length === 0 ? (
-      <Bullseye>{t('Not found')}</Bullseye>
-    ) : (
+  return (
+    <StatusBox
```
Comment (SanjalKatiyar) [inline packages/shared/src/table/selectable-table.tsx:99]: this gonna have conflicts with @alfonsomthd PR, can be update library version in any one of two PRs (we can decide offline in which one) ??

File: packages/mco/components/modals/app-manage-policies/policy-list-view.tsx [hunk 3]
```diff
@@ -184,39 +195,59 @@ export const PolicyListView: React.FC<PolicyListViewProps> = ({
         setModalActionContext={setModalActionContext}
         setMessage={setMessage}
         isActionHidden
+        isSearchHidden={!dataPolicyInfo.length}
```
Comment (SanjalKatiyar) [inline packages/mco/components/modals/app-manage-policies/policy-list-view.tsx:198]: do we need to pass this prop... can we do something like this ??
`dataPolicyInfo.length === 0 ? EmptyState : PolicyListViewToolBar`

File: packages/mco/components/modals/app-manage-policies/helper/policy-list-view-table.tsx [hunk 4]
```diff
@@ -133,6 +133,9 @@ export const PolicyListViewTable: React.FC<PolicyListViewTableProps> = ({
           setModalContext,
         }}
         loaded={true}
+        emptyRowMessage={() => (
+          <Bullseye> {t('No assigned data policy found')} </Bullseye>
+        )}
```
Comment (SanjalKatiyar) [inline packages/mco/components/modals/app-manage-policies/helper/policy-list-view-table.tsx:138]: just a doubt:
we are checking `dataPolicyInfo.length === 0` in `policy-list-view.tsx` file and accordingly displaying table... do we need to pass it here again ??
Is it just for extra safety ??

File: packages/shared/src/table/selectable-table.tsx [hunk 5]
```diff
@@ -97,10 +95,13 @@ export const SelectableTable: SelectableTableProps = <
...
+      loadError={loadError}
+      loaded={loaded}
+      EmptyMsg={emptyRowMessage}
+      data={sortedRows}
```
Comment (alfonsomthd) [inline packages/shared/src/table/selectable-table.tsx:103]: @SanjalKatiyar wouldn't we need to add ` skeleton={<div className="loading-skeleton--table" />}` here?

Comment (SanjalKatiyar) [reply packages/shared/src/table/selectable-table.tsx:103]: we do, but I assumed u will handle it in ur PR (assuming that PR gets merged later) or Gowtham can handle in this PR when he resolves MCs (assuming this PR gets merged later)...

File: packages/mco/components/modals/app-manage-policies/helper/policy-list-view-table.tsx (hunk 4 above)
Comment (GowthamShanmugam) [reply packages/mco/components/modals/app-manage-policies/helper/policy-list-view-table.tsx:138]: dataPolicyInfo is unfiltered, This one is for filtering. 

dataPolicyInfo.length === 0  is for, till now no policy is assigned to the application.
This one is for, policy is assigned but for search text, no matching found.

**Pull request merged**